src/SFML/Window/Unix/KeyboardImpl.cpp

```diff
@@ -481,10 +481,10 @@ void ensureMapping()
             continue;
         }
 
-        std::memcpy(name, descriptor->names->keys[keycode].name, XkbKeyNameLength);
+        std::memcpy(name.data(), descriptor->names->keys[keycode].name, XkbKeyNameLength);
```

Comment:
Kind of wonder whether we can’t just construct a string out of it, instead of doing some memcpy…